src/views/Login.vue

```diff
@@ -1,11 +1,118 @@
 <template lang="pug">
-  p
-    | login works!
+  .auth-container
+    img.logo(src="../assets/wolox_logo.svg" alt="Wolox logo")
+    h2.subtitle
+      | BOOKS
+    form.container-form(@submit.prevent="onSubmit")
+      label.label-input(v-for="field in loginFields" :key="field.id" :for="field.id")
+        | {{ field.label }}
+        input.input-primary(:id="field.id" :type="field.type" v-model="$v.form[field.model].$model")
+        span.alert(v-if="$v.form[field.model].$error")
+          | {{ getError }}
+
+      button.button-primary(type="submit" :disabled="$v.form.$invalid")
+        | Login
+    button.button-secondary(type="button" @click="goSignUp()")
+      | Sign Up
 
 </template>
+
 <script>
+import { loginFields } from './constants'
+import { formErrors } from './../utils/errors'
+import { required } from 'vuelidate/lib/validators'
+import { routes } from '../routes'
+import { getSession } from '../services/users'
+
 export default {
-  name: 'login'
+  name: 'login',
+  data: function () {
+    return {
+      form: {
+        email: '',
+        password: ''
+      },
+      loginFields
+    }
+  },
+  validations: {
+    form: {
+      email: {
+        required
+      },
+      password: {
+        required
+      }
+    }
+  },
+  computed: {
+    getError () {
+      return formErrors.required
+    }
+  },
+  methods: {
+    onSubmit () {
+      let session = { ...this.form }
+      getSession(session)
+        .then(res => console.log(res.data))
+        .catch(err => console.log(err))
+    },
+    goSignUp () {
+      this.$router.push(routes.sign_up)
+    }
+  }
 }
 
 </script>
+
+<style lang="scss" scoped>
+@import "../scss/colors.scss";
+@import "../scss/buttons.scss";
+@import "../scss/inputs.scss";
+
+.container-form {
```

Comment:
Styles are so similar to register component, what do you think use a general styles a use these on inputs.???